pkgs/misc/fastly/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, fetchFromGitHub, installShellFiles, buildGoModule }:
+
+buildGoModule rec {
+  pname = "fastly";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fastly";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "sha256-pMe9hsjxJ4zyzupUWWU2tc5sJb9xhIIdG/Xjh2d7XtY=";
+    # The git commit is part of the `fastly version` original output;
+    # leave that output the same in nixpkgs.
```

Comment:
please just keep a comment that explains it's important delete `.git`

someone could see `leaveDotGit` & deleting `.git` and not understand why we get rid of it

not a big deal though its mostly self explanatory